frontend/web/components/pages/OrganisationSettingsPage.js

```diff
@@ -984,7 +984,10 @@ const OrganisationSettingsPage = class extends Component {
                                                         id='delete-invite'
                                                         type='button'
                                                         onClick={() =>
-                                                          this.deleteUser(id)
+                                                          this.deleteUser(
+                                                            id,
+                                                            `${first_name} ${last_name}`,
```

Comment:
What about just using this message template?
`Are you sure you want to remove the user ${fullname} with email address ${email}`

Reply:
That would still end up looking a bit odd if `fullname` is missing or undefined, right?

Reply:
That would still end up looking a bit odd if `fullname` is missing or undefined, right?

Reply:
I don't think so since (handling the scapes) it would read:
`Are you sure you want to remove the user with email address ${email}`
Which looks proper to me, but ... you know it's not my first language :)

Reply:
I think this is a problem app-wide, I'd suggest we create a format function 

```
const getUserDisplayName = (user?:ProfileSummary)=> {}
```

As for this PR let's use the above for that but create a follow-up ticket investigating where else this can be used.